File: towerOfHanoi.c
void TowersOHanoi(int n, char frompeg, char topeg, char auxpeg)
    {
        if(n == 1) 
        {
            printf("Move disk 1 from peg %c to peg %c",frompeg, topeg);
            return;
        }
        TowersOfHanoi(n-1, frompeg, auxpeg, topeg);
        printf("\n Move disk %d from peg %c to peg %c", n, frompeg, topeg);
        TowersOfHanoi(n-1, auxpeg, topeg, frompeg);
    }   

File: towerOfHanoi.py
def towers_of_hanoi(n, from_peg, to_peg, aux_peg):
    if n == 1:
        print(f"Move disk 1 from peg {from_peg} to peg {to_peg}")
        return
    towers_of_hanoi(n - 1, from_peg, aux_peg, to_peg)
    print(f"Move disk {n} from peg {from_peg} to peg {to_peg}")
    towers_of_hanoi(n - 1, aux_peg, to_peg, from_peg)

# Example usage
towers_of_hanoi(3, 'A', 'C', 'B')
